### ISSUE TITLE
NO_STATUS_CODE_FROM_FUNCTION

### ISSUE THREAD

**Issue by arv** [rocicorp/replicache-sample-todo#9]
I'm getting the following error when accessing https://replicache-sample-todo.now.sh/serve/todo-create:

```
An error occurred with this application.

NO_STATUS_CODE_FROM_FUNCTION
```

Maybe we are not setting the HTTP status correctly? 

![image](https://user-images.githubusercontent.com/45845/81034155-0b35fb80-8e4b-11ea-8c35-62c37cd38bc7.png)

`Authorization` is set to `1`

**Comment by aboodman** [comment]
lol, I didn't actually test it e2e. Of course it doesn't work. Will figure
out.

**Comment by arv** [comment]
Fixed with 7af0b58531fca7b5c85cae3c0a3224bd74e8b7a4
